test-butler-app/src/main/java/com/linkedin/android/testbutler/ButlerService.java

```diff
@@ -56,6 +64,24 @@ public boolean setGsmState(boolean enabled) throws RemoteException {
         public boolean grantPermission(String packageName, String permission) throws RemoteException {
             return permissionGranter.grantPermission(ButlerService.this, packageName, permission);
         }
+
+        @Override
+        public boolean setAccessibilityServiceState(boolean enabled) throws RemoteException {
+            boolean successful = accessibilityServiceEnabler.setAccessibilityServiceEnabled(enabled);
+            if (successful) {
+                accessibilityServiceWaiter.waitForAccessibilityService(enabled);
+            }
+            return successful;
+        }
+
+        @Override
+        public void onDestroy() {
+            super.onDestroy();
+            // Turn the accessibility service off it we enabled it
+            try {
+                accessibilityServiceEnabler.setAccessibilityServiceEnabled(false);
+            } catch (RemoteException ignored) { }
```

Comment:
If you're not putting accessibilityServiceEnabler in ButlerApiStubBase, then IMO this should be moved to onDestroy() below, instead of overriding onDestroy here.

Reply:
That's fair. Will move this.